apollo-federation/src/supergraph/schema.rs

```diff
@@ -99,10 +99,12 @@ pub(crate) fn new_empty_fed_2_subgraph_schema() -> Result<FederationSchema, Fede
 
     directive @federation__listSize(assumedSize: Int, slicingArguments: [String!], sizedFields: [String!], requireOneSlicingArgument: Boolean = true) on FIELD_DEFINITION
 
-    directive @federation__fromContext(field: String) on ARGUMENT_DEFINITION
+    directive @federation__fromContext(field: federation__ContextFieldValue) on ARGUMENT_DEFINITION
```

Comment:
Bugs:
- The type of `@federation__fromContext(field:)` isn't `String`, but should instead be a custom scalar, as it is in the JS.